tests/conftest.py

```diff
@@ -84,7 +84,7 @@ class FuzzyChecker:
     """
 
     def __init__(self) -> None:
-        self.proportion_test_diagnostics = []
+        self.proportion_test_diagnostics: list = []
```

Comment:
See line 189 for where this is used:
```suggestion
        self.proportion_test_diagnostics: list[dict[str, Any]] = []
```